src/main/java/dev/jbang/cli/Alias.java

```diff
@@ -29,6 +31,9 @@ protected Path getCatalog() {
 		if (global) {
 			return Settings.getAliasesFile();
 		} else {
+			if (Files.isDirectory(catalogFile)) {
+				catalogFile = Paths.get(catalogFile.toString(), AliasUtil.JBANG_CATALOG_JSON);
```

Comment:
should it not just return the `Paths.get(...)` result and not change the `catalogFile` ?

Reply:
Indeed :-)

Reply:
Fixed :)